README.md

```diff
@@ -2,20 +2,17 @@
 
 Build status: [![Circle CI](https://circleci.com/gh/NESCent/popgenInfo/tree/master.svg?style=svg)](https://circleci.com/gh/NESCent/popgenInfo/tree/master)
 
-This site provides resources for conducting population genetic analyses in R using existing packages. 
+## Introduction
 
-| [Get started](GETSTARTED.md) | [Learn](LEARN.md)                         |
-| :--------------:|:-------------------------:        |
-| [**Using R**](USINGR.md)     | [**Developer**](DEVELOPER.md) |
-
-
-Any member of the R user community proficient in population genetics is welcome to contibute to this site. See [here](CONTRIBUTING.md) for instructions on how to contribute workflow vignettes using [R markdown](R_MARKDOWN.md).
+This site provides resources for conducting population genetic analyses in R using existing packages. Any member of the R user community proficient in population genetics is welcome to contibute to this site. See [here](CONTRIBUTING.md) for instructions on how to contribute workflow vignettes using [R markdown](R_MARKDOWN.md).
```

Comment:
> Why not just See [instructions](CONTRIBUTING.md) on how to ...?

:+1: 

Good catch. I'm always a fan of reducing the overuse of `[here](link_that_could_go_literally_anywhere)`.